iothub/service/src/HttpRegistryManager.cs

```diff
@@ -1357,7 +1399,7 @@ public override Task<BulkRegistryOperationResult> UpdateTwins2Async(IEnumerable<
         {
             return BulkDeviceOperationsAsync<BulkRegistryOperationResult>(
                 GenerateExportImportDeviceListForTwinBulkOperations(twins, forceUpdate ? ImportMode.UpdateTwin : ImportMode.UpdateTwinIfMatchETag),
-                ClientApiVersionHelper.ApiVersionQueryString,
+                ClientApiVersionHelper.ApiVersionQueryStringDefault,
                 cancellationToken);
```

Comment:
Do we always return default here?

Reply:
even if the env is set to 1?

Reply:
For this method, for sure. We don't want to use the new api-version unless it is the import/export devices job